VK_V1.0/VK_V1.0/Servics/NetworkService.swift

```diff
@@ -81,6 +88,21 @@ final class NetworkService {
         requestData(url: url, completion: completion)
     }
 
+    func fetchNewsfeed(completion: @escaping (ResponseNewsfeed) -> Void) {
+        guard let token = SessionInformation.shared.token else { return }
+        let params = [
+            Constants.versionName: Constants.versionValue,
+            Constants.filtersName: Constants.filtersValue
+        ]
+
+        guard let url: URL = .configureURL(token: token, typeMethod: Constants.newsfeedGetMethod, paramsMap: params)
+        else { return }
+        AF.request(url).responseData { [self] response in
```

Comment:
```suggestion
        AF.request(url).responseData { response in
```
не нашел в документации что требуется вик для AF,  найдешь обратное покажи мне
потом можем пройтись инструментами отладки, поискать лики